SimpleServlet/src/main/java/com/skankhunt220/api/controller/RequestHandler.java

```diff
@@ -0,0 +1,76 @@
+package main.java.com.skankhunt220.api.controller;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.bson.Document;
+
+import main.java.com.skankhunt220.entities.User;
+import main.java.com.skankhunt220.service.UserService;
+
+@WebServlet("/Example") // servlet url
+public class RequestHandler extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+	private UserService userService;
+	private PrintWriter printWriter;
+
+	private void sendResponse(String message, HttpServletRequest request, HttpServletResponse response)
```

Comment:
This method is only responsible for sending a response, so I suggest the following steps:
1. Move the logic for geting `User` fields from the request to a separate method.
2. Send to `Service` methods only those parameters which are really needed and send them in the method where the request came